packages/models-library/tests/test_service_resources.py

```diff
@@ -0,0 +1,57 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from pprint import pformat
+from typing import Type
+
+import pytest
+from models_library.services_resources import (
+    ComposeImage,
+    ImageResources,
+    ResourcesDict,
+    ServiceResources,
+)
+from pydantic import BaseModel, parse_obj_as
+
+
+@pytest.mark.parametrize(
+    "model_cls",
```

Comment:
it is still there. did you really remove them?

Reply:
yes, this was not pushed